specific/cmo_project/models/project.py

```diff
@@ -36,108 +46,377 @@ class ProjectProject(models.Model):
-         ('partner', 'Partner'),
-         ('campaign', 'Campaign'),
-         ],
-        string='Lead Source',
-        states={'close': [('readonly', True)]},
-    )
-    location = fields.Selection(
-        [('local', 'Local'),
-         ('regional', 'Regional'),
-         ('international', 'International'),
-         ],
+    location_id = fields.Many2one(
+        'project.location',
         string='Location',
         states={'close': [('readonly', True)]},
     )
     description = fields.Text(
         string='Description',
+        states={'close': [('readonly', True)]},
     )
-    # Team tab
-    client_service = fields.Many2one(
-        'project.department',
-        string="Client Service",
+    project_from_id = fields.Many2one(
+        'project.from',
+        string='Project From',
         states={'close': [('readonly', True)]},
     )
-    designer = fields.Many2one(
-        'res.partner',
-        string="Designer"
+    project_type_id = fields.Many2one(
+        'project.type',
+        string='Project Type',
+        states={'close': [('readonly', True)]},
     )
-    procurement = fields.Many2one(
-        'res.partner',
-        string="Procurement"
+    project_budget = fields.Float(
+        string='Project Budget',
+        states={'close': [('readonly', True)]},
     )
-    production = fields.Many2one(
-        'res.partner',
-        string="Production"
+    actual_price = fields.Float(
+        string='Actual Price',
+        states={'close': [('readonly', True)]},
     )
-    project_manager = fields.Many2one(
-        'res.partner',
-        string="Product Manager"
+    estimate_cost = fields.Float(
+        string='Estimate Cost',
+        states={'close': [('readonly', True)]},
     )
-    creative = fields.Many2one(
-        'res.partner',
-        string="Creative"
+    pre_cost = fields.Float(
+        string='Pre-Project',
+        states={'close': [('readonly', True)]},
     )
-    graphic = fields.Many2one(
-        'res.partner',
-        string="Graphic"
+    actual_po = fields.Float(
+        string='Actual PO',
+        states={'close': [('readonly', True)]},
     )
-    producer = fields.Many2one(
-        'res.partner',
-        string="Produce"
+    remain_advance = fields.Float(
+        string='Remain Advance',
+        states={'close': [('readonly', True)]},
     )
-    asst_production = fields.Many2one(
-        'res.partner',
-        string="Asst. Production"
-    )
-    # Others Info tab
-    project_from = fields.Selection(
-        [('unknow', 'Unknow'),
-         ('call_in', 'Call in'),
-         ('new_prospect', 'New Prospect'),
-         ('existing_account', 'Existing Account'),
-         ('ceo_office', 'CEO Office'),
-         ],
-        string='Project From',
+    expense = fields.Float(
+        string='Expense',
+        states={'close': [('readonly', True)]},
+    )
+    brief_date = fields.Date(
+        string='Brief Date',
+        default=fields.Date.today,
```

Comment:
Better to use,
default=lambda self: fields.Date.context_today(self),
this will take timezone (from context) into account.
(please fix for all)